profile_page/index.php

```diff
@@ -0,0 +1,86 @@
+<?php
+// Validate is user logged in
+require_once(__DIR__ . '/../validate_user.php');
+
+require_once(__DIR__ . "/../database/repositories/beverages.php");
+require_once(__DIR__ . "/../database/repositories/interests.php");
+require_once(__DIR__ . "/../database/repositories/profile_pictures.php");
+require_once(__DIR__ . "/../database/repositories/users.php");
+require_once(__DIR__ . "/../database/repositories/profiles.php");
+
+$user_ID = (string)get_user_by_credentials($_COOKIE['email'], $_COOKIE['hashed_password'])->fetch_assoc()['id'];
+
+function display_interests(): void
+{
+    global $user_ID;
+    $interests = get_user_interests($user_ID);
+
+    if (is_null($interests)) {
+        echo("No interests :(");
+        return;
+    }
+
+    foreach ($interests as $interest) {
+        echo("<li>" . $interest[0] . "</li>");
+    }
+}
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <?php require_once("../imports.php"); ?>
+    <title>Profile Page</title>
+
+    <style>
+        ul {
+            list-style-type: none;
+            padding-left: 1em
+        }
+    </style>
+</head>
+<body>
+<?php require_once("../nav_bar/index.php"); ?>
+<div class="row row-cols-2 m-3 p-3">
+    <div class="col-4 p-3">
+        <div>
+            <img src="<?= get_user_pfp($user_ID) ?>" alt="Profile Picture" class="rounded-circle img-fluid">
+        </div>
```

Comment:
I agree with the image being square shaped. Generally speaking, the focus of the image is in the centre, I would throw in a fixed width and height so we can prevent something like this:
![image](https://github.com/M-Byte480/Web-Development-CS4116/assets/77645654/3bab67e2-79da-4f09-b15c-63aa1ed6a611)

And object fit to crop it:
```html
         style="width: 100px; height: 100px; object-fit: cover;"
```

And then it comes close to this:
![image](https://github.com/M-Byte480/Web-Development-CS4116/assets/77645654/f3e5ac7c-5598-4e5e-a3d8-f25fb1cb2d89)